gamestonk_terminal/options/yfinance_view.py

```diff
@@ -590,3 +590,66 @@ def show_parity(
         print(show.to_string(index=False))
 
     print("")
+
+
+def risk_neutral_vals(
+    ticker: str, exp: str, put: bool, df: pd.DataFrame, mini: float, maxi: float
+) -> None:
+    """Prints current options prices and risk neutral values [Source: Yahoo Finance]
+
+    Parameters
+    ----------
+    ticker : str
+        Ticker to get expirations for
+    exp : str
+        Expiration to use for options
+    put : bool
+        Whether to use puts or calls
+    df : pd.DataFrame
+        Estimates for stocks prices and probabilities
+    mini : float
+        Minimum strike price to show
+    maxi : float
+        Maximum strike price to show
+    """
+    if put:
+        chain = get_option_chain(ticker, exp).puts
+    else:
+        chain = get_option_chain(ticker, exp).calls
+
+    r_date = datetime.strptime(exp, "%Y-%m-%d").date()
+    delta = (r_date - date.today()).days
+    vals = []
+    rf = get_rf()
```

Comment:
I would allow this to be specified in the controller.

Reply:
The expiration?

Reply:
Ohhhhh. Can do!